src/Sentry.AspNetCore/Sentry.AspNetCore.csproj

```diff
@@ -31,6 +31,13 @@
 
   <ItemGroup Condition="'$(TargetFramework)' != 'netstandard2.0'">
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
+  </ItemGroup>
+
+  <!-- <ItemGroup Condition="'$(TargetFramework)' == 'net6.0'">
+    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="6.0.0" />
+  </ItemGroup> -->
+
```

Comment:
If CI works without this, just delete it:

```suggestion
  <!-- <ItemGroup Condition="'$(TargetFramework)' == 'net6.0'">
    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="6.0.0" />
  </ItemGroup> -->

```

Reply:
Hm, very mysterious. I am wondering how CI can possibly work without that, as it depends on the source generated configuration bindings from `Microsoft.Extensions.Configuration.Binder`:
https://github.com/getsentry/sentry-dotnet/blob/89301cb5b90535e55972bf6aec197ec862afa91c/src/Sentry.AspNetCore/SentryAspNetCoreOptionsSetup.cs#L70

They are only used at design time to generate the `*.g.cs` classes I guess... still doesn't make sense to me though.

Reply:
.NET 6 prob used reflection, and .NET 8 added source generator (get make this trimmable/AOT friendly)

Reply:
That particular method (`Bind`) isnt' available via reflection (it's a generated method) and the codepath that uses it is when `!NETSTANDARD2_0` so it still doesn't compute for me 🤷🏻‍♂️